### ISSUE TITLE
Add photo title to Imgur upload

### ISSUE THREAD

**Issue by JaniL** [FruitieX/teleirc#166]
If the[ Imgur uploading feature](https://github.com/FruitieX/teleirc/pull/165) is enabled, the bot will send the title of the photo aswell with the URL to IRC. (In the same way as with the HTTP server.)

Imgur's image [upload endpoint](https://api.imgur.com/endpoints/image#image-upload) also takes title and description, so it could be included there aswell. However node-imgur's upload method only takes the photo data and Imgur album id as parameters, so currently it's not possible to add those to the uploaded photo.

I've made [a pr](https://github.com/kaimallea/node-imgur/pull/38) to node-imgur, but based on the maintainer's GitHub activity it seems that it might not be merged soon. Options are either using [a fork of node-imgur](https://docs.npmjs.com/files/package.json#git-urls-as-dependencies) as a npm dependency or just wait if it will get merged and updated to npm.

**Comment by Tuupertunut** [comment]
I don't think this is quite as useful, because the titles of images in Telegram are often just parts of a conversation, and don't really mean anything outside of the context.

Example:
```
Tuupertunut: look how ridiculous this new course material is
Tuupertunut: [image], title: like this
```

Now there would be an imgur image with a title "like this".
